packages/web-twig/src/Resources/components/Button/Button.twig

```diff
@@ -11,14 +12,15 @@
 {%- set _rootClassName = _spiritClassPrefix ~ 'Button' -%}
 {%- set _rootBlockClassName = _isBlock ? _spiritClassPrefix ~ 'Button--block' : null -%}
```

Comment:
You can also use `_rootClassName` instead of repeating `Button` with a prefix ;-)

```suggestion
{%- set _rootBlockClassName = _isBlock ? _rootClassName ~ '--block' : null -%}
```